python-service/.env.example

```
@@ -0,0 +1,14 @@
+APP_NAME=Anomaly Python Service
+APP_ENV=development
+APP_HOST=0.0.0.0
+APP_PORT=8090
+MATCH_THRESHOLD=0.78
+LIVENESS_THRESHOLD=0.85
+MAX_ATTEMPTS=3
+SESSION_TTL_MINUTES=15
+MATCH_THRESHOLD=0.78
```

dotenv-linter (4.0.0)

[warning] [DuplicatedKey] The MATCH_THRESHOLD key is duplicated

(DuplicatedKey)